pkg/inctimer/inctimer_test.go

```diff
@@ -21,6 +21,18 @@ import (
 	"time"
 )
 
+func TestTimerAfter(t *testing.T) {
+	for i := 0; i < 100_000; i++ {
+		tr, done := New()
+		select {
+		case <-tr.After(30 * time.Second):
```

Comment:
Why do we need 30s here?

Reply:
Ah, good question - we don't. I changed it to a less arbitrary 1 second instead.

The reason why it was 30 seconds, is because I actually started off the bug hunt by writing this reproduction unit test first, and this is a minimized version of the Hubble Relay unit tests (where we observed the bug initially), which used this timer with a value of 30 seconds:
https://github.com/cilium/cilium/blob/ae37b035d2563b2ae44bfd98fc424ae626700324/pkg/hubble/relay/defaults/defaults.go#L33-L34